libparsec/crates/types/src/addr.rs

```diff
@@ -923,6 +926,59 @@ impl ParsecInvitationAddr {
             .to_http_url(Some(&format!("/invited/{}", &self.organization_id)))
     }
 }
+/*
+ * ParsecAccountEmailValidationAddr
```

Comment:
TODO rename to ParsecAccountActionAddr as per rfc 1013 here https://github.com/Scille/parsec-cloud/pull/10206/files (and it will be more generic